hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-jobclient/src/test/java/org/apache/hadoop/mapred/TestMiniMRBringup.java

```diff
@@ -36,5 +40,21 @@ public void testBringUp() throws IOException {
       if (mr != null) { mr.shutdown(); }
     }
   }
-  
+
+  @Test
+  public void testMiniMRYarnClusterWithoutJHS() throws IOException {
+    MiniMRYarnCluster mr = null;
+    try {
+      final Configuration conf = new Configuration();
+      conf.setBoolean(MiniMRYarnCluster.MR_HISTORY_MINICLUSTER_ENABLED, false);
+      mr = new MiniMRYarnCluster("testMiniMRYarnClusterWithoutJHS");
+      mr.init(conf);
+      Assert.assertEquals(null, mr.getHistoryServer());
```

Comment:
should this assert happen after the call to `start()`?

Reply:
Sure. It makes more stable for future change. I updated the code.